tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "paths": {
       "@kubevirt-utils": ["src/utils"],
       "@kubevirt-utils/*": ["src/utils/*"],
-      "@catalog/*": ["src/views/catalog/*"]
+      "@catalog/*": ["src/views/catalog/*"],
+      "@virtualmachines/*": ["src/views/virtualmachines/*"]
```

Comment:
why we need it? virtualmachines isnt sharable. we just saving some ../../../ ?

Reply:
yes its to avoid ../../......../.. pattern